thirdparty_builtin/CMakeLists.txt

```diff
@@ -57,18 +57,36 @@ if(ENABLE_TESTS)
         list(APPEND gtest_defines "-DGTEST_HAS_DEATH_TEST=0")
     endif()
 
+    #
+    # Use BLT's ENABLE_GMOCK option to set the build options for 
+    # gtest's cmake project
+    #
+    
+    if(ENABLE_GMOCK)
```

Comment:
This off/on and on/off logic doesn't make sense to me. Can you explain? Why does enabling gmock disable building gtest